.github/workflows/operator-integration-test.yml

```diff
@@ -111,8 +111,27 @@ jobs:
           go run validate_instrumentation_vars.go default ns_instrumentation_env_variables.json
           kubectl delete instrumentation sample-instrumentation
 
+      - name: Test for default instrumentation resources for python and java
+        run: |
+          kubectl apply -f integration-tests/python-java/sample-deployment-python-java.yaml
+          sleep 5
+          kubectl get pods -A 
+          kubectl describe pods -n default
+          kubectl describe pods -n amazon-cloudwatch
+          go run integration-tests/manifests/cmd/validate_instrumentation_vars.go default integration-tests/python-java/default_instrumentation_python-java_env_variables.json
+
 
 
+      - name: Test for defined instrumentation resources for python and java
+        run: |
+          kubectl apply -f integration-tests/manifests/sample-instrumentation.yaml
+          sleep 5
```

Comment:
don't we need to restart our pods to check for the new instrumentation values? something like

```
kubectl delete pods --all -n default
kubectl wait --for=condition=Ready pod --all -n default
```

this will delete pods in the default namespace and waits for it to be ready again